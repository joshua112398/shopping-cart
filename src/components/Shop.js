import { Outlet, Link } from 'react-router-dom';
import Cart from './Cart';

function Shop() {

  return (
    <div className='shop'>
      <Cart />
      <div>
        <ul>
          <li><Link to='tops'>Tops</Link></li>
          <li><Link to='jackets'>Jackets and Coats</Link></li>
          <li><Link to='pants'>Pants</Link></li>
          <li><Link to='shorts'>Shorts</Link></li>
          <li><Link to='underwear'>Underwear</Link></li>
          <li><Link to='swimwear'>Swimwear</Link></li>
        </ul>
      </div>
      <Outlet />
    </div>
  );

}

export default Shop;
