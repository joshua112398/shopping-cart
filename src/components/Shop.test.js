import { render } from "@testing-library/react";
import Shop from "./Shop";

describe('Shop', () => {
  test('all links show up on screen', () => {
    const { container } = render(<Shop />);

    expect(container).toMatchSnapshot();
  });

  test('link brings user to the correct page', () => {

  });
});
