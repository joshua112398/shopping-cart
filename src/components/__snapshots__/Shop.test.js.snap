// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shop all links show up on screen 1`] = `
<div>
  <div
    class="shop"
  >
    <div
      data-testid="shopping-cart"
    >
      <h1>
        Your Cart
      </h1>
    </div>
    <div>
      <ul>
        <li>
          <a
            href="/tops"
          >
            Tops
          </a>
        </li>
        <li>
          <a
            href="/jackets"
          >
            Jackets and Coats
          </a>
        </li>
        <li>
          <a
            href="/pants"
          >
            Pants
          </a>
        </li>
        <li>
          <a
            href="/shorts"
          >
            Shorts
          </a>
        </li>
        <li>
          <a
            href="/underwear"
          >
            Underwear
          </a>
        </li>
        <li>
          <a
            href="/swimwear"
          >
            Swimwear
          </a>
        </li>
      </ul>
    </div>
  </div>
  )
</div>
`;
